src/components/table-ajax/table-ajax.stories.js

```diff
@@ -69,7 +72,8 @@ storiesOf('Table Ajax', module)
 
       const pageSize = text('pageSize', '5');
       const paginate = boolean('paginate', TableAjax.defaultProps.paginate);
-      const getCustomHeaders = text('getCustomHeaders');
+      const customHeaders = text('customHeaders', '{ "Accept": "application/json" }');
+      const customHeadersObject = JSON.parse(customHeaders);
```

Comment:
After taking a closer look at the [`@storybook/addon-knobs`](https://www.npmjs.com/package/@storybook/addon-knobs) docs, I've discovered that we can use the [`object`](https://www.npmjs.com/package/@storybook/addon-knobs#object) knob here:

```js
const customHeadersObject = object('customHeaders', { Accept: 'application/json' });
```

Not only is this simpler, but more importantly, it has built-in error-handling for invalid JSON. If the user enters an invalid JSON string, the knob will display a red border to alert the user:

![Error border](https://user-images.githubusercontent.com/50172266/64122717-4ce14480-cd9a-11e9-8248-9fb389b566d8.png)

Otherwise, if we use a `text` knob and `JSON.parse()`, the story will crash and the user will get an unpleasant error screen:

![Error message](https://user-images.githubusercontent.com/50172266/64122210-db54c680-cd98-11e9-9896-137bd5f13f45.png)

Reply:
updated.

Reply:
Thanks :white_check_mark: